javascript-data-model/main.js

```diff
@@ -0,0 +1,60 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
+    orderNum: '114-3941689-8772232',
+    totalPrice: 34.00,
+    orderDate: '2020-08-04',
+    productList: [
+      {
+        productName: 'Javascript for impatient programmers',
```

Comment:
Every item seems to have an image associated with it, but i don't see anywhere that could be pulled from your data model yet 🤔